rock.osdeps

```diff
@@ -235,7 +235,7 @@ dc1394:
 
 vtk:
     debian,ubuntu:
-        '14.04,14.10,15.04,15.10':
+        '14.04,14.10,15.04,15.10,16.04':
```

Comment:
But this conflicts with rock.core osdeps:
vtk-qt4:
  80     ubuntu:
  81       '16.04': [libvtk5-qt4-dev,libvtk5-dev]
  82       default: libvtk5-qt4-dev